src/exporter/exporter/csv.go

```diff
@@ -1,28 +1,69 @@
 package exporter
 
 import (
+	"bufio"
 	"fmt"
-	"github.com/pkg/errors"
-	"os"
+	"sort"
 	"strings"
 
 	"github.com/maxlaverse/blackbox-library/src/blackbox"
+	"github.com/pkg/errors"
 )
 
+var flightModeNames = map[int32]string{
+	1:   "ANGLE_MODE",
+	2:   "HORIZON_MODE",
+	4:   "MAG",
+	8:   "BARO",
+	16:  "GPS_HOME",
+	32:  "GPS_HOLD",
+	64:  "HEADFREE",
+	128: "AUTOTUNE",
+	256: "PASSTHRU",
+	512: "SONAR",
+}
+
+var flightStateNames = map[int32]string{
+	1:  "GPS_FIX_HOME",
+	2:  "GPS_FIX",
+	4:  "CALIBRATE_MAG",
+	8:  "SMALL_ANGLE",
+	16: "FIXED_WING",
+}
+
+var failsafePhaseNames = []string{
+	"IDLE",
+	"RX_LOSS_DETECTED",
+	"LANDING",
+	"LANDED",
+}
+
+var fieldUnits = map[string]string{
+	"time":             "us",
+	"vbatLatest":       "V",
+	"amperageLatest":   "A",
+	"energyCumulative": "mAh",
+	"flightModeFlags":  "flags",
+	"stateFlags":       "flags",
+	"failsafePhase":    "flags",
+}
+
 type CsvFrameExporter struct {
-	target             *os.File
+	target             *bufio.Writer
 	lastSlow           *blackbox.SlowFrame
 	NumberOfFramesRead int
+	debugMode          bool
 }
 
-func NewCsvFrameExporter(file *os.File) *CsvFrameExporter {
+func NewCsvFrameExporter(file *bufio.Writer, debugMode bool) *CsvFrameExporter {
```

Comment:
I just realised, that we can even switch to `io.Writer` interface, because anyway we only write to the target.
It will let us to use any kind of writer, even http response